extensions/cornerstone-dicom-seg/src/getToolbarModule.ts

```diff
@@ -33,6 +65,21 @@ export function getToolbarModule({ servicesManager }: withAppTypes) {
           };
         }
 
+        const activeRepresentations = segmentationService.getSegmentationRepresentations(
```

Comment:
Disable various segmentation tools and utilities based on the type of the active segmentation.